src/Applications/LDAS_App/ens_forcing/enpert_forc.csh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/csh -f 
+
+## env : ADAS_EXPDIR,  GRID,  NENS, RUNDIR
+
+set force_cntr =  "${ADAS_EXPDIR}/recycle/holdforc"
+set force_orig =  "${ADAS_EXPDIR}/atmens"   
+set force_rgd = "${ADAS_EXPDIR}/atmens/rgdlfo" 
+set outgrid =  "${GRID}"
+mkdir $force_rgd
+
+$RUNDIR/regrid_forc.csh  $force_orig $force_rgd $outgrid    
```

Comment:
@saraqzhang, why is `regrid_forc.csh` in $RUNDIR?  Shouldn't this live in `./bin` ?  Any scripts needed by ldas_setup and lenkf.j should be placed into `./bin`.  This probably requires editing `../CMakeLists.txt`.